app/models/course/registration.rb

```diff
@@ -0,0 +1,29 @@
+class Course::Registration < ActiveRecord::Base
+  stampable
+
+  scope :staff, -> { where(role: roles.values_at(:teaching_assistant, :manager, :owner)) }
```

Comment:
Not sure if it the thing Joel wants. Probably he wants a name for the group of enum values? Let's discuss in the next meeting.